include/boost/geometry/core/access.hpp

```diff
@@ -33,10 +33,10 @@ namespace boost { namespace geometry
 {
 
 /// Index of minimum corner of the box.
-int const min_corner = 0;
+inline int const min_corner = 0;
```

Comment:
Can it be made `constexpr`? For me that looks better than an inline global const

Reply:
It could *also* be made `constexpr` if you want. `const` or `constexpr` are the same with regard to linkage. It's the `inline` that is necessary to convert it from internal to external linkage.

Reply:
I should note that the variables here are *implicitly* also `constexpr` because (i) they are of literal type, and (ii) you are providing an initializer. As a consequence, you *can* add `constexpr` to the declaration, but it does not actually change anything.